nexusparser/tools/yaml2nxdl/yaml2nxdl.py

```diff
@@ -108,7 +114,10 @@ def yaml2nxdl(input_file: str, verbose: bool):
         del yml_appdef['category']
```

Comment:
This line and line no: 111 can be removed out of the two if statements. It is always called  anyways.